lib/cogctl/actions/relays/enable.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Cogctl.Actions.Relays.Enable do
```

Comment:
I agree that having them be separate commands (i.e., `cogctl relays enable $FOO` and `cogctl relays disable $FOO` vs. `cogctl relays toggle-activation $FOO enable`) is better UX; I was talking about extracting the implementation logic into one set of functions to reduce the duplication.